packages/playwright/src/makeTest.ts

```diff
@@ -105,6 +101,9 @@ export const makeTest = (
         );
 
         trackComplete();
+
+        // make sure we clear the field associated with this test ID, so the shared chromaticSnapshots object stays small
+        delete chromaticSnapshots[testId];
```

Comment:
I'm not comfortable leaving this line untested, but don't see a way to test this.

We need to make sure we clean up after ourselves on each test, otherwise the `chromaticSnapshots` object could balloon in size and cause an out-of-memory error on the Playwright run.

@tmeasday @tevanoff do you see a feasible way to test the `makeTest()` function? I've been able to satisfactorally test the `takeSnapshot()` changes by using the [Playwright library](https://playwright.dev/docs/library) (instead of the Playwright test runner), but I'm not sure how I'd test `makeTest`. But I think it's important to cover this case.

Reply:
I'm not sure `delete`-ing in this file is super safe, maybe better to export a function that does the deleting from the `takeSnapshot` file might be a little more modular?

--

@skitterm I feel like testing our integration into Playwright from within Playwright is sort of tricky.

My guess is without making things super complicated the only way to do it would be as a unit test -- One way to do it would be to refactor this function out so you can call it directly a couple times (with a mocked `use()` function) and then check the size of the `chromaticSnapshots` object before and after. WDYT?

Reply:
> I'm not sure delete-ing in this file is super safe, maybe better to export a function that does the deleting from the takeSnapshot file might be a little more modular?

@tmeasday I ended up making the object a Map instead here, does that work?

I tried to mock out `page` and such, but things got complicated quickly (since the page creates a `CDPSession` which we then use at length), so I am using the Playwright Library for this test.